hw/misc/aspeed_hace.c

```diff
@@ -104,6 +104,97 @@ static int do_hash_operation(AspeedHACEState *s, int algo)
     return 0;
 }
 
+static int do_hash_sg_operation(AspeedHACEState *s, int algo)
+{
+    hwaddr src, len, dest, buf_len;
+    struct iovec iov[32];
+    unsigned int i = 0;
+    unsigned int is_last = 0;
+    uint8_t *digest_buf = NULL;
+    size_t digest_len = 0;
+    struct aspeed_sg_list *sg_list;
```

Comment:
QEMU likes CamelCase

Reply:
Solved (I guess) in the latest force-push